### PR TITLE
Added platform file and moved docs to assets

### PR DESCRIPTION
**Pull request by mandrew**
Also added robots file and updated url so docs are generated into assets folder

### CODE REVIEW

File: robots.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+User-agent: *
+Disallow: /admin
+Disallow: /assets/src
+Crawl-Delay: 4
```
Comment (camfindlay) [inline robots.txt]: what's crawl delay for?

File: .platform.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+infrastructure: master
```
Comment (camfindlay) [inline .platform.yml]: do we need a platform yaml for virtual stacks?

Comment (assertchris) [inline .platform.yml]: magic 8-ball says; "no".

File: robots.txt (hunk 1 above)
Comment (mandrew) [inline robots.txt]: It basically slows down spiders from hitting your site and using up bandwidth

File: .platform.yml (hunk 2 above)
Comment (mandrew) [inline .platform.yml]: Yep I have a confirmation that it is not needed so will remove

**Pull request merged**